.github/workflows/CI.yml

```diff
@@ -49,7 +49,7 @@ jobs:
     runs-on: macos-latest
     strategy:
       matrix:
-        python-version: ["3.10", "3.11", "3.12"]
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
```

Comment:
Don't you need python 3.8 on macos as well ?

Reply:
Good point; my mind must have been full of Apple Silicon (for which Python gained support in 3.9), but non-`aarch64` would be fine on 3.8 🤔